v23.1/cockroach-debug-encryption-decrypt.md

```diff
@@ -0,0 +1,63 @@
+---
+title: cockroach debug encryption-decrypt
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
```

Comment:
> collecting debug information from all nodes in your cluster.

This command pertains only to the collection of encrypted information from a single node in a cluster.

It must be run locally on the node of interest. It doesn't support connecting as a network client, etc.